docs/mentored_sprint_agenda.md

```diff
@@ -2,35 +2,66 @@
 This event is held one to two weeks following our accessibility learning 
 experience.
 
-## Sprint theme
-Documentation and alt text (media descriptions?)
-
-## Proposed projects
-- [Project Jupyter documentation](https://jupyter.readthedocs.io/en/latest/) 
-and/or 
-- [JupyterLab documentation](https://jupyterlab.readthedocs.io/en/stable/) 
-
-## Event agenda
-- **:00-:10** Folks arrive
-- **:10-:15** goal setting for event/what we plan to do
-- **:15-:30** an icebreaker to introduce attendees to each other
-- **:30-1:00** Learning time.  Present the basics around alt text, the projects, 
-and how the sprint will work. Share resources for further reference in the event.
-- **1:00- 1:05** Team up! Join a room to work on a (pre-determined) section of the docs. 
-- **1:05- 1:45** Work time! There is also a place for review.
-- **1:45- 2:00** Break time! Move around.
-- **2:00- 2:45** Work time part 2! Remember to get review.
-- **2:45- 3:00** Wrap up! Time for giving feedback and info on upcoming events.
-- **3:00- 4:00 Optional** Celebrate! Nominate `alt text hall of fame` by category and vote. 
-Top voted ones and their authors live in infamy on the repo. This time is also spillover 
-for people still wanting to connect with others.
-
-## Questions in search of answers
-- Do we focus only on alt text, or descriptive content for other media as 
-well? The JupyterLab docs have a lot of videos.
-- Do we need resources for git and github, or is that out of scope for our 
-attendees?
-- Project Jupyter docs have less content and are a more manageable goal for 
-the sprint. Do enough people use them for this to do a helpful task?
-- What is a reasonable amount of time for a sprint? How long do we want this 
-to be?
+## October 9
+
+A hands-on event in Jupyter code bases that improve and document accessibility conditions.
+
+### Takeaways
+
+Attendees will become familiar with the different skills needed to impact the Jupyter ecosystem, especially around documentation and alt text.
```

Comment:
improve? positively impact? - for some reason this sounds a bit on the air to me but cannot pint why

Reply:
```suggestion
Attendees will become familiar with the different skills needed to provide a better experience for disabled scientists in the Jupyter ecosystem, especially around documentation and alt text.
```